scripts/generate_evaluator_docs.py

```
@@ -0,0 +1,397 @@
+    # Build the evaluator pages list
+    evaluator_pages = [f"api-reference/evaluators/{slug}" for slug in evaluator_slugs]
+
+    # Find if Evaluators group already exists in navigation
+    navigation = mint.get("navigation", [])
+    evaluators_group = None
+    evaluators_index = None
+
+    for i, group in enumerate(navigation):
+        if isinstance(group, dict) and group.get("group") == "Evaluators":
+            evaluators_group = group
+            evaluators_index = i
```

Ruff (0.14.10)

Local variable `evaluators_index` is assigned to but never used

Remove assignment to unused variable `evaluators_index`

(F841)